tests/test_console.py

```diff
@@ -1,41 +1,20 @@
 import pytest
 from ttt.console import Console
-
-
-class MockConsoleIOErrorInput:
-    def get_input(self, message=""):
-        return "Not a number!"
-
-
-class MockConsoleIOValidInput:
-    def __init__(self):
-        self.last_output = None
-
-    def get_input(self, message=""):
-        return "1"
-
-    def print_output(self, output):
-        self.last_output = output
+from ttt.player import Player
+from ttt.game import Game
+from tests.mocks import MockConsoleIO
```

Comment:
Is there a way you can combine all of the imports from `tests.mocks` into a single import statement?